buckets/archive/tracker.go

```diff
@@ -146,7 +146,7 @@ func (t *Tracker) Track(
 	dbID thread.ID,
 	dbToken thread.Token,
 	bucketKey string,
-	jid ffs.JobID,
+	jid string,
```

Comment:
Same here, not using `ffs` types anymore, but this should just work because `ffs.JobID` is just an alias for `string`.